code/controllers/subsystem/mapping.dm

```diff
@@ -375,6 +375,17 @@ GLOBAL_LIST_EMPTY(the_station_areas)
 	else
 		for(var/M in global.config.maplist)
 			mapvotes[M] = 1
+			
+	var/list/previous_maps = list()
+	if(SSdbcore.Connect())
+		var/datum/DBQuery/query_previous_maps = SSdbcore.NewQuery({"
+			SELECT map_name FROM [format_table_name("round")] WHERE id BETWEEN lower = :lower AND upper = :upper
```

Comment:
In theory, you should check for same port and server IP to handle the possibility of having multiple live servers

Reply:
How would I go about doing this?